packages/api/src/external/ehr/athenahealth/command/get-patient.ts

```diff
@@ -140,6 +161,7 @@ export async function getPatientIdOrFail({
       },
     });
     if (triggerDq) {
+      await sleep(delayBeforeDq.asMilliseconds());
```

Comment:
are we sure that 5 sec is enough for PD to complete?

Reply:
Ironically I just checked and I have fixed the issue where we issue this too quickly already. I'll remove it.